tools/flang1/flang1exe/semfin.c

```diff
@@ -539,7 +539,7 @@ semfin(void)
 
   /* Process data from SAVE statements */
 
-  if (sem.savloc)
+  if (sem.savloc || sem.savall)
```

Comment:
For @gklimowicz's reference. This bug was introduced in commit https://github.com/flang-compiler/flang/commit/d2d64f083b004dc215c052dd22c3108008ee46da. @bryanpkc's change corrects the code here.

Also, note that the same issue reported in your forum.
https://forums.developer.nvidia.com/t/msave-broken-in-ce19-10/136441